index.html

```diff
@@ -132,22 +142,19 @@ <h4>Tonic</h4>
                 <div class="work-description">
                     <h4>Multi-Post Stories</h4>
                     <ul class="description-element">
-                        <li>CANOPY</li>
-                        <li class="element">
-                            <img class="bullet" src="./images/bullet.png" alt="bullet" />
-                        </li>
-                        <li class="element">Back End Dev</li>
-                        <li class="element">
-                            <img class="bullet" src="./images/bullet.png" alt="bullet" />
-                        </li>
-                        <li class="element">2015</li>
+                        <li>FACEBOOK</li>                        
+                        <li> <img class="bullet" src="./images/bullet.png" alt="bullet" /></li>
+                        <li> Full Stack Dev</li>
+                        <li> <img class="bullet" src="./images/bullet.png" alt="bullet" /></li>
+                        <li>2015</li>
                     </ul>
                     <p class="description-text descriptionTxt">
-                        A daily selection of privately personalized reads; no accounts or
-                        sign-ups required.
+                        Experimental content creation feature that allows users to add to an existing story over the course
+                         of a day without spamming their friends.
                     </p>
                     <ul class="languages">
-                        <li class="lang">html</li>                       
+                        <li class="lang">html</li>
+                        <li class="lang3">Ruby on rails</li>
                         <li class="lang1">css</li>
                         <li class="lang2">javaScript</li>
                     </ul>
```

Comment:
- [ ] As requested by the first reviewer, kindly fix the styling of the list of technologies for your project card to match the given template in the mobile view. You are expected ti maintain the design of the mobile view even after adding media queries.
### FIgma 
![image](https://user-images.githubusercontent.com/94551484/221341357-7c6e7195-a2ac-47cb-94f4-e375b2c73b58.png)
### Your design
![image](https://user-images.githubusercontent.com/94551484/221341378-c1cc836a-ed12-48bb-81c3-b2c527002d7d.png)